app/src/main/java/fr/groggy/racecontrol/tv/ui/season/browse/SeasonBrowseFragment.kt

```diff
@@ -49,6 +69,46 @@ class SeasonBrowseFragment : BrowseSupportFragment(), OnItemViewClickedListener
         adapter = eventsAdapter
     }
 
+    /**
+     * Function that will load a [Session.largePictureUrl] as a background image when a card is selected
+     */
+    private fun initializeBackground() {
+        val backgroundManager = BackgroundManager.getInstance(activity).apply {
+            attach(activity?.window)
+        }
+
+        val metrics = DisplayMetrics()
+        requireActivity().windowManager.defaultDisplay.getMetrics(metrics)
+
+        onItemViewSelectedListener = OnItemViewSelectedListener { _, item, _, _ ->
+            if (item is Session) {
+                viewLifecycleOwner.lifecycleScope.launch {
+                    delay(BACKGROUND_UPDATE_DELAY) // A small delay so that the loading of the image and transition is more smooth
```

Comment:
can you elaborate this comment a bit better?

Why is it more smooth ?

Reply:
This will make sure the previous image has finished loading (if that is the case) and the new image will start loading a bit later.I hope it make sense